doc/community.rst

```diff
@@ -1,59 +1,44 @@
 NEST Community
 ===============
 
-
 Mailing List
 -------------
 
 The NEST users mailing list is intended to be a forum for questions on the usage
-of NEST, the exchange of code and general discussions about NEST. The philosophy
+of NEST, the exchange of code and general discussions about NEST.  The philosophy
 is that all users profit by sharing their experience. All NEST core developers
 are subscribed to this list and will participate in the discussions as far as
 time allows.
 
+By subscribing to the mailing list you will also get notified of all NEST related events!
+
 For more information, subscription instructions and access to the archives,
 please visit https://www.nest-initiative.org/mailinglist/.
 
+Open video meeting for users and developers
+--------------------------------------------
 
-Contributing to NEST
----------------------
-
-NEST draws its strength from the many people that use and improve it. We
-are happy to consider your contributions (e.g. own models, bug or
-documentation fixes) for addition to the official version of NEST.
-
-Please see the `NEST developer
-space <http://nest.github.io/nest-simulator>`_ for information about
-the development workflow of NEST and for how to create a fork of our Git
-repository and make pull requests against it.
-
-Reporting bugs
---------------
+Every two weeks, we have an open video meeting to discuss current issues and developments in NEST.
+We welcome our users to join who may have questions regarding their implementations or issues they want help solving.
+This is an opportunity to have discussions in real time with developers.
 
-The primary place to go to if you find an error is the `GitHub issue
-tracker for NEST <https://github.com/nest/nest-simulator/issues>`_.
-Please take the time to check if your issue has already been reported
-there before creating a new one.
+Information for dates and how to join can be found on `our GitHub wiki <https://github.com/nest/nest-simulator/wiki/Open-NEST-Developer-Video-Conference>`_
 
-To make it easier for the developers to understand and solve the
-problem, please include the following information in your bug report, if
-applicable:
+Publications using NEST
+-------------------------
 
-1. Release version or Git revision of NEST.
+We have compiled  a `list of NEST-related peer-reviewed publications <https://www.nest-simulator.org/publications/>`_ that
+we update regularly.
 
-2. Platform and operating system version.
+If you have used NEST in your research, let us know!
+Don't forget to :doc:`cite NEST <citing-nest>` in your work.
```

Comment:
After the change of the Headline in the first line of this PR, should this not rather be the following?
```suggestion
Don't forget to :doc:`cite NEST <cite-nest>` in your work.
```

Reply:
citing-nest is the name of the file and not the header name